test/findex/sqlite_test.dart

```diff
@@ -291,8 +292,27 @@ class SqliteFindex {
         entry.newValue,
         entry.oldValue,
       ]);
+
+      try {
+        final ResultSet resultSet =
+            db.select('SELECT value FROM entry_table WHERE uid=?', [entry.uid]);
+        if (resultSet.isEmpty) {
+          // No rejected value
+          continue;
+        }
+        if (resultSet.length != 1) {
+          throw Exception(
+              "Only 1 entry is expected, found ${resultSet.length} entries");
+        }
+        final Row row = resultSet[0];
+        if (row['value'] == entry.oldValue) {
+          rejectedEntries.add(UidAndValue(entry.uid, entry.oldValue));
```

Comment:
`rejectedEntries` should contains the updated value for the Rust to retry with the most recent change

```suggestion
          rejectedEntries.add(UidAndValue(entry.uid, row['value']));
```

Reply:
But not sure if you result set contains the correct thing here…